src/test/resources/Features/OCD - 2262-RemoveDeprecatedApiPostCallsOfPutOrDelete.feature

```diff
@@ -0,0 +1,49 @@
+Feature: OCD-2262 - Remove API POST calls that were deprecated in favor of PUT/DELETE 
+    This test checks whether deprecated calls are removed from CHPL API webpage
+
+@Regression
+Scenario: CHPL API page loads and shows correct title 
+    Given I am on CHPL API page 
+    Then Page title should be "CHPL API" 
```

Comment:
I'd prefer these steps be written as close to English as possible. So:
```
Given I am on the CHPL API page
Then the page title should be "CHPL API"
```
If you read the steps out loud it should sound like proper English